ImageItem/ImageItem.module.less

```diff
@@ -0,0 +1,28 @@
+// ImageItem.module.less
+//
+
+@import '../styles/skin.less';
+@import '../styles/mixins.less';
+
+.imageItem {
+    .applySkins({
+      .caption {
+        /* Public Class Names */
```

Comment:
Short answer: We don't need it.
Long answer:

We could add the `/* Public Class Names */` when there is nothing in . caption {} to allow for app developers to define some styles in it. In that case, the `publicClassNames` should be defined in a kind. But I don't think that it is your case.

```
	styles: {
		...
		publicClassNames: true
	},
```